### ISSUE TITLE
specify friction parameters in odephysics engine

### ISSUE THREAD

**Issue by rdiankov** [rdiankov/openrave#141]
Nick: http://www.ode.org/ode-latest-userguide.html#const_dContactApprox1

  our variation of the ode plugin is essentially the same as yours, but with more xml tag options
22:49 I think just a contact model tag in the xml would suffice for most people :)
  so users can decide what they want for themselves
 自分: ok, do you have a preferable way you want to specify it?
  or should i decide?
  ..within the xml <contact> field
22:50 Nick: apologies: contact[i].surface.mode = 0 specifies friction force is constant and capped to mu (default in oderave is 1N)
22:51 You can decide - we use multiple <model> tags and just "|" them
  eg:
  <model>dContactApprox1</model>
<model>dContactSoftCFM</model>
<model>dContactSoftERP</model>
  and somehting like:
fricmodel = fricmodel | dContactApprox1;
in our xml parser
22:53 自分: ok, what do you think about this way:
<contact>Approx1|SoftCFM|SoftERP</contact>
  basically remove the dContact stuff
  furthermore, if you want to narrow it down to two bodies only
  you would do
 Nick: sounds good
 自分: <contact body="ground" body="car">Approx1|SoftCFM|SoftERP</contact>
22:54 Nick: that would be good functionality
 自分: i guess we can also add regular expressions
  ie:
22:55 <contact body="ground" body="*">Approx1|SoftCFM|SoftERP</contact>
 Nick: at the moment a typical spec limited between two bodies for us looks like this:
<friction>
<Body>terrain</Body>
<Body>front_right_wheel</Body>
<mu>0.6</mu>
<model>dContactApprox1</model>
<model>dContactSoftCFM</model>
<model>dContactSoftERP</model>
<cfm>0.0001</cfm>
<erp>0.95</erp>
</friction>
